src/mds/Server.cc

```diff
@@ -5794,18 +5794,22 @@ int Server::parse_quota_vxattr(string name, string value, quota_info_t *quota)
       }
     } else if (name == "quota.max_bytes") {
       /*
-       * The "quota.max_bytes" must be aligned to 4MB if greater than or
-       * equal to 4MB, otherwise must be aligned to 4KB.
+       * The "quota.max_bytes" must be aligned to 4MiB if greater than or
+       * equal to 4MiB, otherwise must be aligned to 4KiB.
        */
       string cast_err;
       int64_t q = strict_iec_cast<int64_t>(value, &cast_err);
-      if(!cast_err.empty() ||
-         (!IS_ALIGNED(q, CEPH_4M_BLOCK_SIZE) &&
-          (q < CEPH_4M_BLOCK_SIZE && !IS_ALIGNED(q, CEPH_4K_BLOCK_SIZE)))) {
-        dout(10) << __func__ << ":  failed to parse quota.max_bytes: "
+      if(!cast_err.empty()) {
+        dout(10) << __func__ << ": failed to parse quota.max_bytes: "
                  << cast_err << dendl;
         return -CEPHFS_EINVAL;
       }
+      if (q >= CEPH_4M_BLOCK_SIZE && !IS_ALIGNED(q, CEPH_4M_BLOCK_SIZE) ||
+          q < CEPH_4M_BLOCK_SIZE && !IS_ALIGNED(q, CEPH_4K_BLOCK_SIZE)) {
+        dout(10) << __func__ << ": size must align to 4KiB if less than 4MiB or must align to 4MiB"
+                 << dendl;
```

Comment:
here too,
```
failed to parse quota.max_bytes: size must be aligned to 4KiB if less than 4MiB else it must align to 4MiB

Reply:
I will revise this by:

`failed to parse quota.max_bytes, the value must be aligned to 4KiB if less than 4MiB else it must align to 4MiB`

Reply:
if you intend to add comma instead of colon, then do the same with the above stmt too(at line 5803)

Reply:
All the others are using the format:

`function-name: xxxxx`. I have updated it please check. Thanks.